lib/realtime/channels/cache_supervisor.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Realtime.Channels.CacheSupervisor do
+  @moduledoc """
+  Supervisor for Channels Cache and Operational processes
+  """
+  use Supervisor
+
+  alias Realtime.Channels.Cache
+
+  def start_link(init_arg) do
+    Supervisor.start_link(__MODULE__, init_arg, name: __MODULE__)
```

Comment:
Just curiosity: any reason for having an intermediate supervisor rather than starting Cachex directly in the supervision tree?